README.md

```diff
@@ -83,6 +83,34 @@ $ terraform apply
 
 Run `terraform destroy` when you don't need these resources.
 
+# vpc-flow-logs:
+
+To run this example you need to save this code in Terraform file, and change the values according to our settings.
+
+```hcl
+provider "aws" {
+}
+module "vpc_flow_logs" {
+  source = "coralogix/aws/coralogix//modules/vpc-flow-logs"
+
+  coralogix_region   = "Europe"
+  custom_domain       = "https://<your custom doamin>/api/v1/logs"
```

Comment:
If you have coralogix_region configured no need for custom_domain and vise-versa

It should be explicit, and clear to the user